test/support/src/vfs_helpers.h

```diff
@@ -589,8 +589,8 @@ struct TemporaryDirectoryFixture {
   std::string create_temporary_array(
       std::string&& name, tiledb_array_schema_t* array_schema) {
     auto array_uri = fullpath(std::move(name));
-    REQUIRE_TILEDB_OK(tiledb_array_schema_check(ctx, array_schema));
-    REQUIRE_TILEDB_OK(
+    require_tiledb_ok(tiledb_array_schema_check(ctx, array_schema));
```

Comment:
Could you in this PR or another remove one of REQUIRE_TILEDB_OK/require_tiledb_ok and CHECK_TILEDB_OK/check_tiledb_ok?

Reply:
Done.